scss/components/calendar.scss

```diff
@@ -150,6 +150,8 @@ $fd-calendar-table-width: $fd-calendar-table-item-width * 7 !default;
     height: fd-space(13);
     display: flex;
     align-items: center;
+    width: 276px;
+    margin-left: -12px;
```

Comment:
This _looks_ wrong.  There shouldn't be any reason to force a specific pixel width on the header within a calendar.

Reply:
@greg-a-smith Appropriate changes will be made.